src/components/Template/Template.tsx

```diff
@@ -58,18 +60,6 @@ const GlobalStyles = createGlobalStyle`
     font-family: 'Equinor Mono', source-code-pro, Menlo, Monaco,
     monospace;
   }
-
```

Comment:
Is there a "negative" to having this style global?

Reply:
by "this" style i mean the textfield style*

Reply:
It was overwriting the outline style which was wanted when the textfield component was set to danger/warning